code/__HELPERS/game.dm

```diff
@@ -389,22 +389,19 @@
 				continue
 		spawn(0)
 			G << 'sound/misc/notice2.ogg' //Alerting them to their consideration
-			switch(alert(G,Question,"Please answer in [poll_time/10] seconds!","Yes","No"))
-				if("Yes")
+			switch(askuser(G,Question,"Please answer in [poll_time/10] seconds!","Yes","No", StealFocus=0, Timeout=poll_time))
+				if(1)

```

Comment:
no defines.

the number is the number of the button that was selected, if You make me define it i'll just switch it back to returning the button text.